db/schema.rb

```diff
@@ -10,13 +10,13 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20180703023011) do
+ActiveRecord::Schema.define(version: 20180828084425) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
   enable_extension "uuid-ossp"
 
-  create_table "activities", force: :cascade do |t|
+  create_table "activities", id: :serial, default: %q{nextval('activities_id_seq'::regclass)}, force: :cascade do |t|
```

Comment:
This `id: :serial` thing doesn't work. Just commit the part of the diff that has to do with your migration.